projects/kit/components/input-phone-international/tokens/countries-masks.ts

```diff
@@ -70,16 +71,18 @@ export const TUI_COUNTRIES_MASKS: InjectionToken<Record<TuiCountryIsoCode, strin
                 [TuiCountryIsoCode.FJ]: `+679##-#####`,
                 [TuiCountryIsoCode.FK]: `+500#####`,
                 [TuiCountryIsoCode.FM]: `+691###-####`,
-                [TuiCountryIsoCode.FR]: `+33 ## ## ## ## ##`,
+                [TuiCountryIsoCode.FR]: `+33(0) # ## ## ## ##`,
```

Comment:
why `(0)`

Reply:
As I said in the PR description ⬆️ : 

French numbers always start with a 0 and although it is valid to omit it when giving the telephone area code, some services do not work without it. This is the case with Firebase, for example, which does not send SMS if the 0 is missing.

`+33612345678` and `+330612345678` are both valid but `+33612345678` does not always work and people tend to omit the 0 when `+33` is displayed so the input displays `+33 61 23 45 67 8`. 
Starting with the 0 and asking for a single character after (which can be either 6 or 7) seems clearer to me.

(+33 0# ## ## ## ## would have been better but this mask does not work while +33(0) # ## ## ## ## does.)

It's an opinionated change that you can dismiss.

Reply:
thank you